include/Tableau.hpp

```diff
@@ -0,0 +1,120 @@
+/*
+* This file is part of the MQT QMAP library which is released under the MIT license.
+* See file README.md or go to https://iic.jku.at/eda/research/ibm_qx_mapping/ for more information.
```

Comment:
the link in the header still points to a JKU site. It would be great if you could swap it out with the corresponding cda-tum link.